deps/llvm.mk

```diff
@@ -135,6 +135,14 @@ LLVM_CMAKE += -DLLDB_DISABLE_PYTHON=ON
 endif # LLDB_DISABLE_PYTHON
 endif # BUILD_LLDB
 
+# Part of the FreeBSD libgcc_s kludge
+ifeq ($(OS),FreeBSD)
+ifneq ($(GCCPATH),)
+LLVM_CMAKE += -DCMAKE_INSTALL_RPATH="\$$ORIGIN:$(GCCPATH)"
```

Comment:
which cmake dependencies don't currently set this? maybe cleaner to always do in CMAKE_COMMON on freebsd, if that doesn't hurt anything on its own?